iac-extensions/kubernetes/src/test/resources/checks/CpuLimitCheck/test_pod_object.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: example
+spec:
+  containers:
+    - name: noncompliant_without_cpu_limit # Noncompliant {{Specify a CPU limit for this container.}}
+    # ^^^^
+      image: nginx
+      resources:
+        limits:
+          memory: 64Mi
+
+    - name: noncompliant_without_limits # Noncompliant
+    # ^^^^
+      image: nginx
+      resources:
+        requests:
+          memory: 64Mi
+
+    - name: noncompliant_without_resources # Noncompliant
+    # ^^^^
+      image: nginx
+
+    - name: noncompliant_cpu_limit_tilde
+      image: nginx
+      resources:
+        limits:
+          cpu: "~" # Noncompliant
+        # ^^^^^^^^
+
+    - name: noncompliant_cpu_limit_empty_brackets
+      image: nginx
+      resources:
+        limits:
+          cpu: "[]" # Noncompliant
+        # ^^^^^^^^^
+
+    - image: nginx
+      name: noncompliant_cpu_limit_null
+      resources:
+        limits:
+          cpu: "null" # Noncompliant
+        # ^^^^^^^^^^^
+
+    - name: compliant_cpu_limit # Compliant
+      image: nginx
+      resources:
+        limits:
+          cpu: 100m
```

Comment:
I think in ITS I saw a case with `"0.5"` as the argument (double quoted), Can you add this also as a test case?

Reply:
In JIRA as well as RSPEC it is 0.5 without double quote. And in [K8s docs](https://kubernetes.io/docs/tasks/configure-pod-container/assign-cpu-resource/), I just found the case with double quote